cmd/kind/delete/cluster/deletecluster.go

```diff
@@ -49,6 +49,15 @@ func NewCommand() *cobra.Command {
 }
 
 func runE(flags *flagpole) error {
+	// Check if the cluster name exists
+	known, err := cluster.IsKnown(flags.Name)
```

Comment:
this doesn't need to check, just delete? maybe inform?

Reply:
it should not check. delete should always try to delete and clean up kubeconfig